packages/playground/windows/playground/packages.config

```diff
@@ -2,6 +2,5 @@
 <packages>
   <package id="Microsoft.Windows.CppWinRT" version="2.0.210312.4" targetFramework="native"/>
   <package id="Microsoft.UI.Xaml" version="2.5.0" targetFramework="native"/>
-  <package id="Microsoft.WinUI" version="3.0.0-preview4.210210.4" targetFramework="native"/>
```

Comment:
Looks like this was removed by accident? @chiaramooney

Reply:
@asklar autolinking will automatically remove this if you do a `run-windows` in playground.